src/App/docutils/ComponentPreview/index.js

```diff
@@ -161,31 +161,42 @@ const ComponentPreview = ({ children, language, removeOuterTag, hideValue, remov
 
             this.state = {
                 activeTab: this.props.showCasePanelAdvanced.elements[0],
-                optionsOpen: false,
+                optionsOpen: window.innerWidth > 1200, // XL grid breakpoint
                 activeOptions: this.props.showCasePanelAdvanced.elements[0].activeOptions ? [...this.props.showCasePanelAdvanced.elements[0].activeOptions] : []
             };
         }
 
+        _resetOptions () {
+            const showcasePanel = document.getElementById(this.props.showCasePanelAdvanced.id);
+
+            showcasePanel.querySelectorAll("input[type=checkbox]").forEach(checkbox => checkbox.checked = false);
+            showcasePanel.querySelectorAll("select[id]").forEach(dropdown => dropdown.value = 0);
+            showcasePanel.querySelectorAll(".radio").forEach(radio => radio.querySelector("input").checked = false);
+
+            const defaultRadios = [...showcasePanel.querySelectorAll(".radio")].filter(radio => radio.querySelector("input").id.includes("default"));
+
+            if (defaultRadios.length > 0) {
+                const defaultRadiosGroups = defaultRadios.map(radio => radio.querySelector("input").name);
+
+                defaultRadios.map(radio => radio.querySelector("input").checked = true);
+                [...showcasePanel.querySelectorAll(".radio")].filter(radio => !defaultRadiosGroups.includes(radio.querySelector("input").name))
+                    .map(radio => radio.querySelector("input").checked = radio.querySelector("input").value === "0");
```

Comment:
Very nice! I'd love a comment with a short explanation for these chained operations so that one does not have to do these in their heads to figure out what exactly the purpose is.

Other than that, great :)